src/module_3/module_3.py

```diff
@@ -0,0 +1,129 @@
+# %% Import packages
+import pandas as pd
+
+from sklearn.model_selection import train_test_split
+from sklearn import linear_model
+from sklearn.linear_model import Ridge
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.ensemble import RandomForestClassifier
+from sklearn import metrics
+from sklearn.preprocessing import normalize
+
+import matplotlib.pyplot as plt
+
+# %% Load data
+address = "D:\\Yo\\"
```

Comment:
In general, in a Python script ever add code outside a function (except global variable declarations). So create a function (for example `def train_model()`) and include the logic there. In order to make the script executable as a program (for example from the console) include `if __name__=='__main__` at the end ([check this out](https://medium.com/@mycodingmantras/what-does-if-name-main-mean-in-python-fa6b0460a62d#:~:text=In%20this%20example%2C%20the%20main,function%20to%20execute%20the%20code) for more details)